PowerEditor/src/NppCommands.cpp

```diff
@@ -3048,6 +3048,13 @@ void Notepad_plus::command(int id)
 			}
 		}
         break;
+		
+		case IDM_LANG_OPENUDLDIR: {

```

Comment:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#good

Reply:
And the wrong indentation.